.github/workflows/check-lint-udeps-test.yaml

```diff
@@ -0,0 +1,133 @@
+# Basic flow where we do all generic tests:
+#   - `cargo check`: Check that our crate compiles
+#   - `cargo fmt --all -- --check`: Checks that our crate follows basic Rust formatting
+#   - `cargo workspaces exec cargo +nightly udeps`: Makes sure that the branch does not have any unused dependencies
```

Comment:
Shouldn't we also mention the `exec cargo verify-project` job here?

Reply:
Done in https://github.com/tchataigner/mock-crate/pull/4/commits/e5838d54f7ddd240fbce5064cfe812d0400af348